### PR TITLE
Integrate RN Nightly Build 7/4

### PR DESCRIPTION
**Pull request by chiaramooney**
### Type of Change
_Erase all that don't apply._
- Integration

### What
https://github.com/facebook/react-native/compare/bcc69dfb3...68f3a42fc

Commits:
- https://github.com/facebook/react-native/commit/7b0ba6d94f9a325efdded1d2a1727c833b8f97f4
- https://github.com/facebook/react-native/commit/5854b11bf9d42bab9dbe62b9152a3d3a94e42c13
- react-native-community/cli 9.0.0

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows/pull/10241)

### CODE REVIEW

File: packages/@office-iss/react-native-win32/src/Libraries/Pressability/Pressability.win32.js [hunk 1]
```diff
@@ -286,20 +286,20 @@ const Transitions = Object.freeze({
   },
 });
 
-const isActiveSignal = signal =>
+const isActiveSignal = (signal: TouchState) =>
```
Comment (aeulitz) [inline packages/@office-iss/react-native-win32/src/Libraries/Pressability/Pressability.win32.js:289]: For my education: Type annotation looks like TypeScript - should this be reflected in the file name extension?

Comment (aeulitz) [reply packages/@office-iss/react-native-win32/src/Libraries/Pressability/Pressability.win32.js:289]: Nevermind, this is probably Flow.

**Comment by chiaramooney** [comment]
Clang issue still present

**Review by acoates-ms — APPROVED** [review]

File: vnext/ReactCommon/ReactCommon.vcxproj.filters [hunk 2]
```diff
@@ -151,7 +151,6 @@
       <Filter>yoga\event</Filter>
     </ClCompile>
     <CLCompile Include="$(ReactNativeDir)\ReactCommon\reactperflogger\reactperflogger\BridgeNativeModulePerfLogger.cpp" />
-    <ClCompile Include=".\Yoga.cpp" />
```
Comment (asklar) [inline vnext/ReactCommon/ReactCommon.vcxproj.filters:154]: how come we're not including yoga here? 🤔

**Review by asklar — APPROVED** [review]

**Pull request merged**